Testreglar/2.4.5/Nett/nett-2.4.5a.json

```diff
@@ -0,0 +1,654 @@
+			"ruting": {
+				"ja": {
+					"type": "ikkjeForekomst",
+					"utfall": "Nettstaden har berre ei nettside."
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Finst det ei liste med lenker til innhaldet på nettstaden?",
+			"ht": "<p>Gjer ein visuell inspeksjon og sjå etter ei liste med lenker. Det kan for eksempel vere</p>\n<ul>\n<li>ein meny</li>\n<li>ei innhaldsliste</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.4"
+				}
+			}
+		},
+		{
+			"stegnr": "2.4",
+			"spm": "Har nettstaden ein søkefunksjon?",
+			"ht": "<p>Gjer ein visuell inspeksjon og sjå etter søkefelt.</p>\n<p><strong>Merk:</strong></p>\n<ul>\n<li>Søkefunksjonen skal ligge på den aktuelle nettstaden, og ikkje på ein annan nettstad som Google.</li>\n<li>Nettstaden kan bruke eksterne søk, men søkefunksjonen skal uansett ligge på nettstaden.</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.5"
+				}
+			}
+		},
+		{
+			"stegnr": "2.5",
+			"spm": "Har nettstaden eit nettstadkart?",
+			"ht": "<p>Gjer ein visuell inspeksjon og sjå etter eit nettstadkart. </p>",
+			"type": "jaNei",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.6"
+				}
+			}
+		},
+		{
+			"stegnr": "2.6",
+			"spm": "Har nettstaden andre måtar å navigere på, for eksempel ei alfabetisk sideoversikt?",
+			"ht": "<p>Søkefelt på same side eller lenke til søk.</p>\n<p><strong>Merk:</strong></p>\n<ul>\n<li>Søkefunksjonen skal ligge på den aktuelle nettstaden og ikkje på ein annan nettstad, soml Google.</li>\n<li>Nettstaden kan bruke eksterne søk, men søkefunksjonen skal uansett ligge på nettstaden.</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"alle": {
+					"type": "regler",
+					"regler": {
+						"1": {
+							"sjekk": [
+								"2.3",
+								"2.4",
+								"2.5",
+								"2.6"
+							],
+							"type": "talDersom",
+							"verdi": "Ja",
+							"mellom1": 2,
+							"mellom2": 4,
+							"handling": {
+								"type": "gaaTil",
+								"steg": "3.1"
+							}
+						},
+						"2": {
+							"sjekk": [
+								"2.3",
+								"2.4",
+								"2.5",
+								"2.6"
+							],
+							"type": "talDersom",
+							"verdi": "Ja",
+							"mellom1": 0,
+							"mellom2": 1,
+							"handling": {
+								"type": "avslutt",
+								"fasit": "Nei",
+								"utfall": "Nettstaden har ikkje minst to måtar å navigere på."
+							}
+						}
+					}
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Kva nettside testar du?",
+			"ht": "<p>Oppgi URL eller Side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side-ID:",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Er nettsida del av ein prosess, der kvar enkelt handling er nødvendig for å gjennomføre ein aktivitet?",
```

Comment:
Er det ikke mer hensiktsmessig å sjekke om det er en prosess i starten? Da slipper du å gå gjennom hele testregelen for å komme til dette.

Reply:
Steg 3.2 gjelder selve nettside som egentlig er først steg rett etter nettsiden registreres. Alle 2.X steg gjelder nettsted, så det er ikke mulig å flytte dette steget i tidligere i testregelen.